app/Http/Controllers/CalendarsController.php

```diff
@@ -149,8 +147,6 @@ public function destroy(DeleteCalendarRequest $request, $calendarId)
             return response()->json(['message' => 'De kalender werd niet verwijderd, er is iets foutgegaan.'], 400);
```

Comment:
Make message translatable

Reply:
Again, comes from legacy code. Making text translatable will be handled in a different issue.

Reply:
will be done in issue  https://github.com/StadGent/laravel_site_opening-hours/issues/85